src/AbstractState.cpp

```diff
@@ -1064,7 +1064,7 @@ CoolPropDbl AbstractState::calc_second_partial_deriv(parameters Of1, parameters
 
 #ifdef ENABLE_CATCH
 
-#include "catch.hpp"
+#include <catch2/catch_all.hpp>
```

Comment:
Fixup includes. Technically we could use a more granular header such as `<catch2/catch_test_macros.hpp>`, but I didn't bother (link time isn't such an issue anyways)